public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -68,7 +68,7 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
         >
           {scopes && !meta.dashboardNotFound && <scopes.Component model={scopes} />}
           <NavToolbarActions dashboard={model} />
-          {controls && hasControls && (
+          {controls && (
```

Comment:
this is needed so that the dash controls are activated, otherwise we will no be able to handle the hidden refresh picker manual activation via controls activation handler.

Reply:
@ivanortegaalba - is this ok to do? Given https://github.com/grafana/grafana/pull/88920

Reply:
That should be ok, since it returns null the renderer in case this is not met.

I wonder if the padding is added.

To test this properly, we should check:
- it doesn't add an extra padding when all controls are deselected. This can be tested by playing a playlist.
- The default home doesn't show controls

Reply:
@ivanortegaalba  - I think both of the cases you have mentioned are not having any extra padding :)